webapp/components/product-info.js

```diff
@@ -40,7 +41,7 @@ const versionPatterns = Object.freeze({
 });
 
 // The set of all browsers known to the wpt.fyi UI.
-const AllBrowserNames = Object.freeze(['chrome', 'deno', 'edge', 'firefox', 'flow', 'safari', 'servo', 'webkitgtk', 'android_webview']);
+const AllBrowserNames = Object.freeze(['chrome_android', 'chrome', 'deno', 'edge', 'firefox', 'flow', 'safari', 'servo', 'webkitgtk', 'android_webview']);
```

Comment:
put this after `chrome`

Reply:
We need put this one before 'chrome'. AllBrowserNames is used to create some regular expression, as chrome is a prefix of chrome_android, we need put chrome_android in front of chrome.

Reply:
I can update this to make it alphabetic order also.